tools/cloud/demo.sh

```diff
@@ -424,6 +466,9 @@ echo ${DECODER_MANIFEST_ARN}
 
 echo "Adding DBC signals to decoder manifest..."
 DBC=`cat hscan.dbc | base64 -w0`
+# For MAC users, run this command instead:
```

Comment:
You could use `uname` to auto detect whether host is Mac or Linux.

Reply:
thanks, will use it